CodePush.m

```diff
@@ -362,11 +366,19 @@ - (void)savePendingUpdate:(NSString *)packageHash
 /*
  * This is native-side of the RemotePackage.download method
  */
-RCT_EXPORT_METHOD(downloadUpdate:(NSDictionary*)updatePackage
+RCT_EXPORT_METHOD(downloadUpdate:(NSDictionary*)immutableUpdatePackage
                         resolver:(RCTPromiseResolveBlock)resolve
                         rejecter:(RCTPromiseRejectBlock)reject)
 {
     dispatch_async(dispatch_get_main_queue(), ^{
+        NSDictionary* updatePackage = [immutableUpdatePackage mutableCopy];
+        NSUserDefaults *preferences = [NSUserDefaults standardUserDefaults];
```

Comment:
Why are we pulling this from the defaults as opposed to just getting the current binary timestamp and storing that in the package?